tlsf_cpp/include/tlsf_cpp/tlsf.hpp

```diff
@@ -137,4 +142,24 @@ constexpr bool operator!=(
   return a.memory_pool != b.memory_pool;
 }
 
+template<typename T, size_t PoolSize>
+rcl_allocator_t get_rcl_allocator(tlsf_heap_allocator<T, PoolSize> allocator)
```

Comment:
Needs to be in the `rclcpp::allocator` namespace?

Reply:
Argument-dependent lookup saves us: https://en.cppreference.com/w/cpp/language/adl. As long as the function is in the namespace of one of its arguments, the overload will be resolved.